includes/discovery/sensors/gw-eydfa.inc.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * gw-eydfa.inc.php
+ *
+ * LibreNMS temperature discovery module for Glass Way WDM EYDFA Optical Amplifier
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2018 TheGreatDoc
+ * @author     TheGreatDoc
+ */
+
+$oids = snmp_walk($device, 'oaPumpTable', '-Osq', 'NSCRTV-HFCEMS-OPTICALAMPLIFIER-MIB');
+d_echo($oids."\n");
+
+if ($oids) {
+    echo 'GW EYDFA PUMP ';
+}
+
```

Comment:
For all of these below where you would do multiple snmp gets for the high/low values. I would do a `if (is_numeric($value)) {` check before them so that if the value isn't returned for whatever reason then we save on a bunch of gets.